xml/System.Web.UI.DataVisualization.Charting/SeriesChartType.xml

```diff
@@ -14,7 +14,14 @@
   </Base>
   <Docs>
     <summary>Specifies a chart type for a <see cref="T:System.Web.UI.DataVisualization.Charting.Series" />.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>
+<format type="text/markdown"><![CDATA[  
+## Remarks
+
+For more information about each chart type please refere to our [detail documention](https://docs.microsoft.com/en-us/previous-versions/visualstudio/visual-studio-2012/dd489233(v%3dvs.110)).
```

Comment:
Some comments:
- there should be a comma after "chart type"
- please refere to our --> see 
- rather than refer to it as "detail documentation", just include the title: "\[Chart Types]"